config/config_test.go

```diff
@@ -107,6 +108,111 @@ func TestSampleConfig(t *testing.T) {
 	assertBoolsEqual(t, "metrics.prometheus.enabled", cfg.Metrics.Prometheus.Enabled, true)
 }
 
+func TestNewConfigFuncFileParam(t *testing.T) {
```

Comment:
Can we delete the function `newViperFromSample` and its corresponding test `TestSampleConfig`? I don't think we need this now that we're reading in sample yaml config files.

Reply:
Do you want me to put everything inside `const sampleConfig` into a file instead?

Reply:
I don't think so. I'd rather you keep what you have. I don't see a need for that function nor do I like the go string constant to define a sample yaml file.

Reply:
Removed